test/worker/convert/test_pdf2tifs.py

```diff
@@ -0,0 +1,33 @@
+import os
+import shutil
+
+import unittest
+from pathlib import Path
+
+from worker.convert.converter import convert_pdf_to_tifs
+
+
+class Pdf2TifsTest(unittest.TestCase):
```

Comment:
Was für die Tasknamen gilt, gilt auch für die Klassennamen. Können wir uns auf 3 Buchstaben für Formate einigen?

Reply:
Und wo wir schon dabei sind, sollten wir auch die Verwendung von "to" / "2" vereinheitlichen. Ich plädiere für "to".